creator-node/src/components/healthCheck/healthCheckComponentService.js

```diff
@@ -29,7 +33,52 @@ const healthCheck = async ({ libs } = {}, logger, sequelize) => {
   // we have a /db_check route for more granular detail, but the service health check should
   // also check that the db connection is good. having this in the health_check
   // allows us to get auto restarts from liveness probes etc if the db connection is down
-  await sequelize.query('SELECT 1')
+  const databaseLiveness = await getMonitor(MONITORS.DATABASE_LIVENESS) === 'true'
+  if (!databaseLiveness) throw new Error('Database connection failed')
+
+  return response
+}
+
+/**
+ * Perform a verbose health check, returning health check results
+ * as well as location info, and system info.
+ * @param {*} ServiceRegistry
+ * @param {*} logger
+ */
+const healthCheckVerbose = async ({ libs } = {}, logger, getMonitor = getMonitorValue) => {
+  const basicHealthCheck = await healthCheck({ libs }, logger, getMonitor)
+
+  // Location information
+  const country = config.get('serviceCountry')
+  const latitude = config.get('serviceLatitude')
+  const longitude = config.get('serviceLongitude')
+
+  // System information
+  const databaseConnections = parseInt(await getMonitor(MONITORS.DATABASE_CONNECTIONS))
```

Comment:
can we Promise.all these?

Reply:
good callout, i think these are cheap enough that it wouldn't make a material difference since js is single-threaded, but it looks like ioredis has a pipeline function that can accept multiple commands and do one back & forth, so looking into that

Reply:
if these calls are cheap enough, I think we can merge as is for now!

Reply:
Check it out! New change uses redis.pipeline :-) Ran through same tests. Adding @SidSethi  to take another look as well

Reply:
oh this is very cool

Reply:
naiseeeee